lua/lspconfig/fsautocomplete.lua

```diff
@@ -0,0 +1,61 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local bin_path = 'C:/tools/fsac/fsautocomplete.dll'
+if vim.fn.has('win32') == 0 then
+  bin_path = os.getenv('HOME')..'/lib/fsac/fsautocomplete.dll'
+end
+
+configs.fsautocomplete = {
+  default_config = {
+    cmd = {'dotnet', bin_path, '--background-service-enabled'};
```

Comment:
I would just document what this command has to be, and let people populate `cmd`, see sumneko_lua for inspiration